README.md

```diff
@@ -18,9 +18,14 @@ Built on top of [tui-rs](https://github.com/fdehau/tui-rs) to create the termina
 
 # Installation
 You can use the [cargo][cargo] package manager in order to install it.
+
 ```
 $ cargo install termchat
 ```
+On linux `libxkbcommon` is required (in order to stream videos).
+
+You can install it using your distribution package manager, example on ubuntu: `sudo apt-get install libxkbcommon-dev`
```

Comment:
Could you add a minor comment about the current issue (only receive but not send) in Windows/MacOs

Reply:
https://github.com/lemunozm/termchat/pull/24/commits/bdd4d44eb8fa42687a7df786617c30d52733c932